### ISSUE TITLE
Duplicate static elements

### ISSUE THREAD

**Issue by pepebe** [modxcms/revolution#13454]
### Summary
If you (quick) duplicate a chunk (or any other element, for example templates, snippets, plugins, etc) that is bound to a static file on your webserver you'll get two static chunks targeting the same file. 

Every modification to the new chunk will also effect the original chunk and vice versa.

This might not cause problems if you are using normal edit, but inside a quick edit field you don't see that the file you are editing is in fact a static file.

The current procedure can cause A LOT OF PAIN in a few seconds (yes this happened to me this week). 

### Step to reproduce
Duplicate a static element

### Observed behavior
Duplicating a static element results in two static elements targeting the same file.

### Expected behavior
There is room for discussion here. 

From the top of my hat I'd say during the duplicate process it would be a wise idea to copy the content of the static file into the content field and remove the static property. 

It would also be nice to get some kind of visual feedback that helps to differentiate static elements from db elements. This is especially important for quick edit and would also be useful if it could be added to the tree.

### Environment
Any.

**Comment by mindeffects** [comment]
👍
+999 from me, cause this once killed hours of my work! :-(

**Comment by pepebe** [comment]
The same for me... :(

**Comment by Jako** [comment]
1. would be my preferred solution, since it could be easier changed back to static afterwards.

But VersionX (if that works with static elements) and a file backup could help too.

**Comment by mrhaw** [comment]
Not saying I have an opinion but for the sake of adding to the discussion:
**3. Warn the user upon duplicating that the element is static**

**Comment by Mark-H** [comment]
Agreed with @Jako - 1 sounds best. This has also bitten me in the past :P

**Comment by mindeffects** [comment]
I tend to **option 3**, too, because having a choice is great. If I develop and use static files, there is a reason why I do this. If MODX would duplicate my files into the db, then I would have to revert this every time. A nagger.
In a perfect world: I am asked if I eant to have the duplicate stored in the db or as a file – and samte time have a system setting with the options "ask","file" and "db". This way nobody feels patronizes can configure the system in his favorite way.

**Comment by Mark-H** [comment]
It's already clear that the element is static based on the icon, or if you're editing it, by the static checkbox being shown along with the path, so I don't think a warning is necessary. 

If it needs to be configurable, just put the is_static checkbox and static_file fields in the duplicate window so it can be set/changed right there. Otherwise, the next thing you'll get is requests to change the file/path name by which a new static file is created.

**Comment by mindeffects** [comment]
Good thinking, @Mark-H! Still there must be a check for *static_field* and an "no-way-dude", if the file already exists.
